src/App.js

```diff
@@ -1,8 +1,27 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState } from 'react';
+
 
 const App = () => {
+
+  const [chatEntries, setChatsLike] = useState(chatMessages)
+
+  const updateLikeData = updatedChat =>{
```

Comment:
Rather than receiving a new message with it's like status flipped here, rewrite this to accept an id. The logic about how to copy the message and flip the liked status is better housed here, since the state _must_ have all the information necessary to make a new message, whereas in complex systems, we may not pass all of the information need to make a new data element all the way down to a presentation component like ChatEntry.